### PR TITLE
⭐️ add query pack to count assets in env

### PR DESCRIPTION
**Pull request by atomic111**

### CODE REVIEW

**Review by chris-rock — APPROVED** [review]
Yeah!!!

File: extra/mondoo-asset-count.mql.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - uid: mondoo-asset-count
+    name: Asset Count Query Pack
+    queries:
+      - uid: mondoo-asset-count-on-vsphere-cluster-esxi
+        title: Retrieve all ESXi hosts
+        filters: asset.platform == 'vmware-vsphere'
+        query: |
+          vsphere.datacenters { hosts.length }
+      - uid: mondoo-asset-count-on-vsphere-cluster-vms
+        title: Retrieve all VMs from vsphere cluster
```
Comment (tas50) [inline extra/mondoo-asset-count.mql.yaml:11]: vSphere

File: extra/mondoo-asset-count.mql.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        filters: asset.platform == 'azure'
+        query: |
+          azure.compute.vms.length
+      - uid: mondoo-count-users-in-azure
+        title: Retrieve all VMs from Azure
+        filters: asset.platform == 'azure' || asset.platform == 'microsoft365'
+        query: |
+          azuread.users.length
+      - uid: mondoo-asset-count-on-aws
+        title: Retrieve all VMs from AWS
```
Comment (tas50) [inline extra/mondoo-asset-count.mql.yaml:26]: Instances not VMs in AWS

File: extra/mondoo-asset-count.mql.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        filters: asset.platform == 'vmware-vsphere'
+        query: |
+          vsphere.datacenters { vms.length }
+      - uid: mondoo-asset-count-on-azure
+        title: Retrieve all VMs from Azure
+        filters: asset.platform == 'azure'
+        query: |
+          azure.compute.vms.length
+      - uid: mondoo-count-users-in-azure
+        title: Retrieve all VMs from Azure
```
Comment (tas50) [inline extra/mondoo-asset-count.mql.yaml:21]: The query is users but the text is VMs

File: extra/mondoo-asset-count.mql.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        filters: asset.platform == 'azure' || asset.platform == 'microsoft365'
+        query: |
+          azuread.users.length
+      - uid: mondoo-asset-count-on-aws
+        title: Retrieve all VMs from AWS
+        filters: asset.platform == 'aws'
+        query: |
+          aws.ec2.instances.length
+      - uid: mondoo-asset-count-in-windows-domain
+        title: Retrieve all computer object from the windows domain
```
Comment (tas50) [inline extra/mondoo-asset-count.mql.yaml:31]: Windows

**Pull request merged**